src/AWS.Distro.OpenTelemetry.AutoInstrumentation/OtlpAwsSpanExporter/OtlpAwsSpanExporter.cs

```diff
@@ -0,0 +1,346 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Please add this line as well since we have code that is based on those written by otel. 
```
// Modifications Copyright The OpenTelemetry Authors. Licensed under the Apache License 2.0 License.
```